homeassistant/components/v2c/config_flow.py

```diff
@@ -0,0 +1,53 @@
+"""Config flow for V2C integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pytrydan import Trydan
+from pytrydan.exceptions import TrydanError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+    }
+)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for V2C."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        errors: dict[str, str] = {}
+        if user_input is not None:
+            evse = Trydan(user_input[CONF_HOST])
```

Comment:
Why don't we use the HA httpx client here?

Reply:
indeed 🤦